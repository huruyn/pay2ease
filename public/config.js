window.API_URL = "https://zippy-retail-api-dev.azurewebsites.net";