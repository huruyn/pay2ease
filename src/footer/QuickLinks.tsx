export default function QuickLinks() {
  return (
    <div className="quick-links container">
      <div className="row">
        <h3 className="col-sm-12">Quick Links</h3>
      </div>
      <div className="row">
        <div className="col-sm-2">
          <ul>
            <li>
              <a href="/" onClick={(e) => e.preventDefault()} className="text-decoration-none">Home</a>
            </li>
            <li>
              Please Review Our Legal Stuff by visiting our
              {' '}
              <a href="/legal" onClick={(e) => e.preventDefault()} className="text-decoration-none">Privacy Policy | TOU | EULA | CardHolder T&amp;C Pages</a>
            </li>
          </ul>
        </div>
        <div className="col-sm-2">
          8465 W Sahara Ave.
          Las Vegas, NEVADA
          <ul>
            <li>
              <a href="tel:+17026231876">1.702.623.1876</a>
            </li>
            <li>
              <a href="mailto:contact@zippy.cash">contact@zippy.cash</a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  );
}
