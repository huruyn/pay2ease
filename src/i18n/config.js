import i18n from 'i18next';
import { initReactI18next } from 'react-i18next';

i18n.use(initReactI18next).init({
  fallbackLng: 'en-US',
  lng: 'en-US',
  resources: {
    'en-CA': {
      translations: require('./locales/en-CA/translations.json')
    },
    'en-US': {
      translations: require('./locales/en-US/translations.json')
    },
    'es-US': {
      translations: require('./locales/es-US/translations.json')
    },
    'fr-CA': {
      translations: require('./locales/fr-CA/translations.json')
    },
  },
  ns: ['translations'],
  defaultNS: 'translations'
});

i18n.languages = ['en-CA', 'en-US', 'es-US', 'fr-CA'];

export default i18n;