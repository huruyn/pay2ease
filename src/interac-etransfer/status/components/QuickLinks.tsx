export default function QuickLinks() {
  return (
    <div className="quick-links">
      <div className="notify-safe">
        <i className="zippy-cash-icon zc-security" />
        Online Security Guarantee
      </div>
      <h3>You can also</h3>
      <ul>
        <li>View upcoming bill payments and transfers</li>
        <li>Register for InteracÂ® e-Transfer Autodeposits</li>
        <li>Change your statement preferences</li>
      </ul>
    </div>
  );
}
