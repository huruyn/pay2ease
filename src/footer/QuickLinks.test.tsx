import '../i18n/config';
import QuickLinks from './QuickLinks';

const ReactTestRenderer = require('react-test-renderer');

describe('QuickLinks Component', () => {
  it('matches the snapshot', () => {
    const tree = ReactTestRenderer.create(<QuickLinks />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
