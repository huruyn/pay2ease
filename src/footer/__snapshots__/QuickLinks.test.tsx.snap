// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuickLinks Component matches the snapshot 1`] = `
<div
  className="quick-links container"
>
  <div
    className="row"
  >
    <h3
      className="col-sm-12"
    >
      Quick Links
    </h3>
  </div>
  <div
    className="row"
  >
    <div
      className="col-sm-2"
    >
      <ul>
        <li>
          <a
            className="text-decoration-none"
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li>
          Please Review Our Legal Stuff by visiting our
           
          <a
            className="text-decoration-none"
            href="/legal"
            onClick={[Function]}
          >
            Privacy Policy | TOU | EULA | CardHolder T&C Pages
          </a>
        </li>
      </ul>
    </div>
    <div
      className="col-sm-2"
    >
      8465 W Sahara Ave. Las Vegas, NEVADA
      <ul>
        <li>
          <a
            href="tel:+17026231876"
          >
            1.702.623.1876
          </a>
        </li>
        <li>
          <a
            href="mailto:contact@zippy.cash"
          >
            contact@zippy.cash
          </a>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
